src/TestSuite/Fixture/TestFixture.php

```diff
@@ -285,6 +297,65 @@ public function insert(ConnectionInterface $db)
     }
 
     /**
+     * {@inheritDoc}
+     */
+    public function createConstraints(ConnectionInterface $db)
+    {
+        if (empty($this->_constraints)) {
+            return true;
+        }
+
+        foreach ($this->_constraints as $name => $data) {
+            $this->_schema->addConstraint($name, $data);
+        }
+
+        $sql = $this->_schema->addConstraintSql($db);
+
+        if (empty($sql)) {
+            return true;
+        }
+
+        try {
+            foreach ($sql as $stmt) {
+                $db->execute($stmt)->closeCursor();
+            }
+        } catch (\Exception $e) {
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function dropConstraints(ConnectionInterface $db)
+    {
+        if (empty($this->_constraints)) {
+            return true;
+        }
+
+        $sql = $this->_schema->dropConstraintSql($db);
+
+        if (empty($sql)) {
+            return true;
+        }
+
+        try {
+            foreach ($sql as $stmt) {
+                $db->execute($stmt)->closeCursor();
+            }
+        } catch (\Exception $e) {
+            debug($e);
```

Comment:
We might not want to catch these errors. Letting them bubble up alerts the developer of an issue with their constraints/schema.